.circleci/config.yml

```diff
@@ -45,6 +45,31 @@ jobs:
       - restore_cache:
           key: vendor-{{ checksum "Gopkg.lock" }}
       - run: go build -o machine-controller github.com/kubermatic/machine-controller/cmd/controller
+      - save_cache:
+          key: machine-controller-{{ .Revision }}
+          paths:
+            - machine-controller
+
+  publish:
+    machine: true
+    steps:
+      - run: >
+          sudo mkdir -p /go/src/github.com/kubermatic/machine-controller &&
```

Comment:
sudo should not be required. If its required, we a producing this binary with wrong permissions

Reply:
It is not required because of the binary, it is required to create the folders